test/test_process_nrl.py

```diff
@@ -0,0 +1,59 @@
+"""
+Regression test for the process_NRL.py script
+"""
+
+
+def test_process_sample_nrl_file():
+    """
+    This test will download a sample file from S3 and process the file, then compute the SHA-256
```

Comment:
I mentioned git-lfs in my comment but this is actually the cheaper way to go - storing it on S3 and grabbing it when needed.